content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Notation v1.0.0-RC.4 is available!
+author:  "Notation Release Team"
+date: 2023-04-21
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of Notation CLI v1.0.0-rc.4, notation-go v1.0.0-rc.4, and notation-core-go v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds the following significant enhancements:
+
+- Support validating certificate revocation with [Online Certificate Status Protocol](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol) (OCSP)
+- Introduce switch `NOTATION_EXPERIMENTAL=1` to enable experimental features
+- Introduce new CLI command `notation policy` to simplify trust policy configuration
+- Support OCI distribution referrers API
+- Introduce signing and verification with [OCI image layout](https://github.com/opencontainers/image-spec/blob/v1.0/image-layout.md) as experimental feature
+
+### Support validating Certificate revocation with OCSP
+
+OCSP stands for Online Certificate Status Protocol. It is a protocol used to check the revocation status of digital certificates. When a certificate is revoked, it means that it is no longer valid and should not be trusted. To enforce validating certificate revocation, the verification level of trust policy configuration should be set to `strict` as following. Once this is done, the CLI command `notation verify` can be used to validate the signatures including checking certificate revocation status with OCSP.
```

Comment:
Do we need to mention that as long as one of the certificates in the cert chain is detected as revoked, we fail the verification? (given `strict` trust policy configuration)

Reply:
yes, I can add the result, if revocation is detected, what the result will be